build.gradle

```diff
@@ -19,20 +21,24 @@ buildscript {
       bugsnag: 'com.bugsnag:bugsnag:2.0.0',
       intellij: "IC-$ideaVersion",
       jps: "JPS-$ideaVersion",
-      retrolambda: 'me.tatarka:gradle-retrolambda:3.3.1',
       okio: 'com.squareup.okio:okio:1.11.0',
       intellijCore: fileTree(dir: 'lib/intellij-core/lib', include: ['*.jar']),
       sqlitePsi: fileTree(dir: '/Users/astrong/Development/sqlite/sqlite-psi/core/build/libs', include: ['*.jar']),
+      supportSqlite: "android.arch.persistence.room:db:$roomVersion",
+      supportSqliteImpl: "android.arch.persistence.room:db-impl:$roomVersion",
+      robolectric: "org.robolectric:robolectric:3.4.2",
     ]
 
     isCi = "true".equals(System.getenv('CI'))
   }
   repositories {
     mavenCentral()
+    google()
     jcenter()
   }
 
   dependencies {
+    classpath dep.androidPlugin
```

Comment:
unfortunately this requires android studio to work now. Means that the grammar kit plugin isnt installable among other things probablu